README.md

```diff
@@ -18,7 +18,7 @@ Add [`rc`] to your autoload or source [`connect.kak`] and its [modules] manually
 ### Add to desktop
 
 1. Copy [`kakoune-connect.desktop`] to `$XDG_DATA_HOME/applications`.
-2. Copy [`kakoune-icon.png`] to `$XDG_DATA_DIRS/icons`. Note that `$XDG_DATA_DIRS` is a list of folders.
+2. Copy [Kakoune's logo](https://github.com/mawww/kakoune/blob/master/doc/kakoune_logo.svg) to `$XDG_DATA_HOME/icons/hicolor/scalable/apps/kakoune.svg`.
```

Comment:
What is `hicolor`?

Reply:
> this part of the documentation clearly says that we should locate our icons in the default hicolor them.

It's the default color scheme. It is present on systems supporting xdg. Applications that don't change their icon according to the theme put their logos there.